src/patternfly/components/ContextSelector/context-selector.scss

```diff
@@ -299,6 +298,14 @@
   background-color: var(--pf-c-context-selector__menu--BackgroundColor);
   background-clip: padding-box;
   box-shadow: var(--pf-c-context-selector__menu--BoxShadow);
+
+  &:only-child,
```

Comment:
do we still need this if we're going to add the modifier in react? I was thinking the class would let us get around needing to use this markup dependent selector.